packages/ui/src/common/components/selects/SimpleSelect.tsx

```diff
@@ -1,8 +1,8 @@
-import { isDefined } from '@/common/utils'
 import React, { useEffect, useReducer, useState } from 'react'
 import styled, { css } from 'styled-components'
 
 import { Colors } from '../../constants'
+import { isDefined } from '../../utils'
```

Comment:
Okay I changed the path because I had got a [`Cannot find module '@/common/utils' from 'src/common/components/selects/select.tsx'`](https://github.com/Joystream/pioneer/runs/2503331652?check_suite_focus=true) error on build. But I had not pulled the main branch yet (so maybe the `tsconfig.json` was not up to date on this branch)